src/Storages/MergeTree/MergeTreeIndexFullText.cpp

```diff
@@ -748,7 +748,7 @@ MergeTreeIndexGranulePtr MergeTreeIndexFullText::createIndexGranule() const
     /// have an old index.
     /// TODO: remove this at the end of 2024.
     if (index.type == INVERTED_INDEX_NAME)
-        throw Exception(ErrorCodes::ILLEGAL_INDEX, "Indexes of type 'inverted' are no longer supported. Please drop and recreate the index as type 'full-text'");
+        throw Exception(ErrorCodes::ILLEGAL_INDEX, "Indexes of type 'inverted' are no longer supported. Please drop and recreate the index as type 'full_text'");
```

Comment:
Using full-text doesn't work and returns the following error message

```
Syntax error: failed at position 55 (''full-text''):

ALTER TABLE events_v2 ADD INDEX ft_idx(pathname) TYPE 'full-text';

Expected one of: expression with optional parameters, function, function name, compound identifier, identifier
```
or without quotes
```
Syntax error: failed at position 59 ('-'):

ALTER TABLE events_v2 ADD INDEX ft_idx(pathname) TYPE full-text;

Expected one of: token sequence, Dot, token, OpeningRoundBracket, GRANULARITY, FIRST, AFTER, Comma, INTO OUTFILE, FORMAT, SETTINGS, end of query
```